spring-beans/src/main/java/org/springframework/beans/factory/xml/BeanDefinitionParserDelegate.java

```diff
@@ -1406,7 +1406,7 @@ public BeanDefinitionHolder decorateIfRequired(
 					return decorated;
 				}
 			}
-			else if (namespaceUri.startsWith("http://www.springframework.org/")) {
+			else if (namespaceUri.startsWith("http://www.springframework.org/schema/")) {
```

Comment:
Can you explain why that change is part of the nohttp check? Not denying that's useful but I expect something like this in a dedicated issue.

Reply:
It seems the tool ignores "http://www.springframework.org/schema/" prefixes in general since that's used by XML namespace declarations, so it automatically also ignores that Java source bit if its String contains that full prefix.

Reply:
Yes. The tooling ignores well known XML names. For Spring that is it starts with http://www.springframework.org/schema/ and does not end with `.xsd`